product_docs/docs/pgd/5/durability/legacy-sync.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Legacy synchronous replication using PGD
+navTitle: Legacy synchronous replication
+---
+
+!!! Warning Important
+Consider using [Group Commit](group-commit) instead.
+!!!
```

Comment:
I wonder if this still is true. Or if we can refer to the commit scope based Sync Rep option, instead. Do we have that documented by now?

I guess it's good enough for this PR and we can revisit Sync Rep independent of this PR.

Reply:
We don't have it documented yet, was waiting on details from Petr. The next iteration on this doc would be to bring in Sync Rep.